src/commands/connections/util.ts

```diff
@@ -0,0 +1,32 @@
+export const outputFormat = {
+	table: [
+		{
+			header: 'ID',
+			property: 'id'
+		},
+		{
+			header: 'Name',
+			property: 'name'
+		},
+		{
+			header: 'Platform',
+			property: 'platform.name'
+		},
+		{
+			header: 'Enabled',
+			property: 'disabled',
+			transform: val => !val
+		},
+		{
+			header: 'IsPool',
+			property: 'group'
+		},
+		{
+			header: 'ParentPool',
+			property: 'pool.id'
```

Comment:
I would show pool name here and pool id in the json below

Reply:
Or at least show pool name in the json

Reply:
Made the change, but it appears the REST service is returning connection.Name in the connection.Pool.Name field. Looking into that now...

Reply:
Data reference between connection & pool was using the wrong column. Fixed server-side, no client changes needed